docs/tutorials/data/hess.ipynb

```diff
@@ -22,31 +22,54 @@
   {
```

Comment:
<p>A few comments on the implementation:</p><ul><li>I think we should use cutouts here, otherwise the code will not be usable for larger maps</li><li>Just use the higher level API i.e.  <code>MapDatasetMaker.make_exposure</code></li><li> You can change the unit if a Map like <code>Map.to_unit()</code></li></ul>
<br/>

 _Reply via <a href='https://app.reviewnb.com/gammapy/gammapy/pull/3511/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='16'/>